src/test/java/com/zufar/onlinestore/exception/handler/JwtTokenExceptionsHandlerTest.java

```diff
@@ -0,0 +1,28 @@
+package com.zufar.onlinestore.exception.handler;
+
+import com.zufar.onlinestore.security.exception.JwtTokenException;
+import com.zufar.onlinestore.security.exception.handler.JwtTokenExceptionsHandler;
+import org.junit.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
```

Comment:
IMO, those tests are useless. Expecially the testing of SpringSecurityConfig. It makes sence if your security configuration contains complex conditional logic that can lead to different authentication providers or different security configurations being set up under different conditions, etc. It doesnt contain any assertions, verifications logic , so I believe we dont need it